aggregateLogs_CFS.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+#
+#This script is simply collection of the commands located in the Openshift 3.0 Administration Guide for
+# aggregating logs using the "Centralized File System" approach
+#
+#
+#USAGE: ./aggregateLogs_CFS.sh 
+#
+#
+
+
+function error_check {
+
+    if [[ $1 -ne 0 ]];then
+       echo "Error during $2 exit_code: ${status}"
+       exit status
```

Comment:
also, not sure status variable will be available inside of funtion. If you are passing it i, you need to set it locally via status=$1.